cookbooks/kinematically_driven_subduction_2d/doc/kinematically_driven_subduction_2d.md

```diff
@@ -100,3 +99,98 @@ boundary. This can also be seen in {numref}`fig:QQ_case1_results`.
 
  Case 1 viscosity snapshots at 8 and 15 My.
 ```
+## Case 2a: Model with thermal evolution
+
+Case 2a builds on Case 1 by including a heterogeneous initial temperature field
+that combines the plate cooling model for the temperature in the plates with an
+adiabatic temperature gradient below the plates. For the plate cooling model,
+we assume that the overriding plate is 40 My old and the subducting plate 70 My.
+The temperature function is too complex to comfortably write out as a function
```

Comment:
may be explain how this initial temperature field is built ? it was a summation of many terms, wasn't it ?
Then the comment about it being 'complex' makes a bit more sense.
Also I am not sure about 'comfortably'.